lib/out/static-output.js

```diff
@@ -9,23 +9,26 @@ function uppercaseFirstLetter(str) {
     return str[0].toUpperCase() + str.substr(1);
 }
 
-function render(pkg) {
+function render(pkg, currentState) {
     const packageName = pkg.moduleName;
     const rows = [];
 
     const indent = '           ' + emoji('   ');
+    const flags = currentState.get('global') ? '--global' : `--save${pkg.devDependency ? '-dev' : ''}`;
+    const upgradeCommand = `npm install ${flags} ${packageName}@${pkg.latest}`;
+    const updateMessage = `${chalk.green(upgradeCommand)} to go from ${pkg.installed} to ${pkg.latest}`;
```

Comment:
small nit but should it be `upgradeMessage`? To align with `upgradeCommand` and `easyUpgrade`...